### ISSUE TITLE
gdpr-cookie-consent-banner with SvelteKit

### ISSUE THREAD

**Issue by zofiaifoz** [beyonk-adventures/gdpr-cookie-consent-banner#25]
Does the consent banner work with Sveltekit? I tried to implement it but I get this error:

SyntaxError: The requested module '/node_modules/@beyonk/gdpr-cookie-consent-banner/src/util.js' does not provide an export named 'validate'

**Comment by zofiaifoz** [comment]
OK @antony  I'll see what I can do. thx.

**Comment by jcamilleri13** [comment]
I'm also trying to get this working on SvelteKit. Would you be interested in a PR if we could get it to play nice @antony?

**Comment by antony** [comment]
Hey @jcamilleri13 - We'd absolutely love a PR!

**Comment by jcamilleri13** [comment]
Ok, I think I've got it working - #28.
I've tested it on my own SvelteKit project and it works nicely. There's been a minor change to the exported bundles, but I think it might be because I compiled them with a different minor Svelte version. Nothing seems drastic enough to affect functionality.

The `cookie-universal` dependency isn't ESM-compatible yet, so that will also need to be updated, but I've included information on a workaround in the README.
